rest/config.go

```diff
@@ -1023,9 +1023,28 @@ func RunServer(config *ServerConfig) {
 	}
 
 	sc := NewServerContext(config)
-	for _, dbConfig := range config.Databases {
+	uuids := make(map[string][]string)
+
+	for dbName, dbConfig := range config.Databases {
 		if _, err := sc.AddDatabaseFromConfig(dbConfig); err != nil {
-			base.Fatalf("Error opening database %s: %+v", base.MD(dbConfig.Name), err)
+			base.Fatalf("Error opening database %s: %+v", base.MD(dbName), err)
+		}
+		dbContext, err := sc.GetDatabase(dbName)
+		if err != nil {
+			base.Fatalf("Error getting database context %s: %+v", base.MD(dbName), err)
+		}
+		if !dbContext.BucketSpec.IsWalrusBucket() {
+			uuid, err := dbContext.Bucket.UUID()
+			if err != nil {
+				base.Warnf("Error getting Bucket UUID %s: %+v", base.MD(dbName), err)
+			}
+			if value := uuids[uuid]; value != nil {
+				base.Warnf("Caution: Duplicate buckets detected in configuration; database %v shares the same bucket("+
```

Comment:
- Shorten log message
- Log bucket name (Bucket UUID won't mean much to a user)
- Prefer `%s` or `%q` (quoted strings) over `%v`
- Redact args as metadata

```go
base.Warnf("Unsupported config: Database %q shares bucket %q with databases %s",
	base.MD(dbName), base.MD(dbContext.Bucket.GetName()), base.MD(value))
```

Reply:
Fixed in latest revision.